static/app/components/noProjectMessage.spec.tsx

```diff
@@ -132,4 +132,22 @@ describe('NoProjectMessage', function () {
       screen.getByText('You need at least one project to use this view')
     ).toBeInTheDocument();
   });
+
+  it('shows projects to superusers if membership is not required', function () {
+    ProjectsStore.loadInitialData([
+      TestStubs.Project({hasAccess: true, isMember: false}),
```

Comment:
do we always return `hasAccess: true` for projects when you're superuser?

Reply:
No. The component I'm modifying has a prop `superuserNeedsToBeProjectMember` that if that is set they will only see projects that have access to. And if false they see all projects. This component is used in multiple places so that's why the prop exist. Let me remove the access field from the test to make it simpler since it doesn't matter.